docs/tasks/debug-application-cluster/audit.md

```diff
@@ -208,153 +127,50 @@ rules:
-```
+In both cases, audit events structure is defined by the API in the
+`audit.k8s.io` API group. The current version of the API is
+[`v1beta1`][auditing-beta-api].
 
-With legacy format, events are formatted as follows:
+### Log backend
 
-```
-2017-09-05T06:08:19.885328047-04:00 AUDIT: id="c28a95ad-f9dd-47e1-a617-b6dc152db95f" stage="RequestReceived" ip="172.16.116.128" method="list" user="kubecfg" groups="\"system:masters\",\"system:authenticated\"" as="<self>" asgroups="<lookup>" namespace="default" uri="/apis/rbac.authorization.k8s.io/v1/namespaces/default/roles" response="<deferred>"
-2017-09-05T06:08:19.885328047-04:00 AUDIT: id="c28a95ad-f9dd-47e1-a617-b6dc152db95f" stage="ResponseComplete" ip="172.16.116.128" method="list" user="kubecfg" groups="\"system:masters\",\"system:authenticated\"" as="<self>" asgroups="<lookup>" namespace="default" uri="/apis/rbac.authorization.k8s.io/v1/namespaces/default/roles" response="200"
-```
+Log backend writes audit events to a file in JSON format. You can configure
+log audit backend using the following [kube-apiserver][kube-apiserver] flags:
 
-Logged events omit the request and response bodies. The `Request` and
-`RequestResponse` levels are equivalent to `Metadata` for legacy format. This legacy format
-of advanced audit is different from the [Legacy Audit](# Legacy Audit) discussed above, such
-as changes to the method values and the introduction of a "stage" for each event.
+- `--audit-log-path` specifies the log file path, that log backend uses to write
+  audit events. Not specifying this flag disables log backend.
+- `--audit-log-maxage` defined the maximum number of days to retain old audit log files.
+- `--audit-log-maxbackup` defines the maximum number of audit log files to retain.
+- `--audit-log-maxsize` defines the maximum size of the audit log file before it gets rotated.
 
-#### Webhook backend
+### Webhook backend
 
-The audit webhook backend can be used to have [kube-apiserver][kube-apiserver]
-send audit events to a remote service. The webhook requires the `AdvancedAuditing`
-feature flag and is configured using the following command line flags:
+Webhook backend sends audit events to a remote API, which is assumed to be the
+same API as [kube-apiserver][kube-apiserver] exposes. You can configure webhook
+audit backend using the following kube-apiserver flags:
 
-```
---audit-webhook-config-file=/etc/kubernetes/audit-webhook-kubeconfig
---audit-webhook-mode=batch
-```
-
-`audit-webhook-mode` controls buffering strategies used by the webhook. Known modes are:
-
-- `batch` - buffer events and asynchronously send the set of events to the external service.
-- `blocking` - block API server responses on sending each event to the external service.
+- `--audit-webhook-config-file` specifies the path to a file with a webhook
+  configuration. Webhook configuration is effectively a [kubeconfig][kubeconfig].
+- `--audit-webhook-mode` define the buffering strategy, one of the following:
```

Comment:
Should we use `--audit-webhook-buffered`  default is true?

Reply:
Let's not deprecate this flag in this release, since it's too close. It'll be a TODO for the next release